js/test/test.js

```diff
@@ -109,31 +109,33 @@ if (settings && settings.skip) {
 
 //-----------------------------------------------------------------------------
 
-async function test (methodName, exchange, ... args) {
-    console.log ('Testing', exchange.id, methodName, '(', ... args, ')')
+async function tester_func (methodName, exchange, ... args) {
```

Comment:
`test` was too generic function name. for a while, let it be something other (even `tester_func`) which will is of more distinctive name